app/controllers/partners/profiles_controller.rb

```diff
@@ -9,12 +9,18 @@ def update
         flash[:success] = "Details were successfully updated."
         redirect_to partners_profile_path
       else
+        handle_social_media_error_message if current_partner.errors.include?(:no_social_media_presence)
```

Comment:
@cielf This conditional statement prevents the social media error from rendering if other validations fail but the social media section is valid. I'm aware that there are no other validations on this edit page, but if any are added in the future something like the following could happen without this conditional statement:

![Screenshot (569)](https://user-images.githubusercontent.com/42154066/194166211-defa2abe-9ea6-45a9-8cf5-48e8f98c3a33.png)

Reply:
> @cielf This conditional statement prevents the social media error from rendering if other validations fail but the social media section is valid. I'm aware that there are no other validations on this edit page, but if any are added in the future something like the following could happen without this conditional statement....

It seems to me that if the social media section is valid, then  current_partner.errors.include?(:no_social_media_presence) should be false.       Is that not so?

Reply:
@cielf That's correct, it would be false.